src/olympia/editors/helpers.py

```diff
@@ -53,8 +53,8 @@ def file_review_status(addon, file):
             return _(u'Pending Preliminary Review')
     # Special case: prelim upgrading to full approval,
     # file can already be preliminary reviewed or not
-    if (file.status in [amo.STATUS_LITE, amo.STATUS_UNREVIEWED]
-            and addon.status == amo.STATUS_LITE_AND_NOMINATED):
+    if (file.status in [amo.STATUS_LITE, amo.STATUS_UNREVIEWED] and
+            addon.status == amo.STATUS_LITE_AND_NOMINATED):
```

Comment:
Is it just me being in Monday-mode or how can this if statement make any sense? If the `addon.status` is in `(lite, unreviewed)` it can't be `== lite_and_nominated` can it?